aspnetcore/blazor/tutorials/build-a-blazor-app.md

```diff
@@ -174,6 +174,10 @@ cd TodoList
 
 1. When finished, shut down the app in the command shell. Many command shells accept the keyboard command <kbd>Ctrl</kbd>+<kbd>C</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>C</kbd> (macOS) to stop an app.
 
+## Publish to Azure
+
+For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/quickstart-dotnetcore).
```

Comment:
Doesn't title need to change? No SQL now

Reply:
Ugh.  Yes, of course.  ouch.  Thanks.